src/ProjectTemplates/Web.ProjectTemplates/content/SignalR-CSharp/wwwroot/js/serverTime.js

```diff
@@ -0,0 +1,17 @@
+var clockConnection = new signalR.HubConnectionBuilder().withUrl("/streamingtime").build();
```

Comment:
Can we use async/await in javascript yet?

Reply:
removing this part of the template per feedback from @Tratcher and @anurse to make it simpler to consume by users.

Reply:
My comment applies to the other piece of code too

Reply:
In the past, we have gotten feedback on docs that use `async`/`await` that it doesn't work in downlevel browsers (because of course it doesn't). We should at least put a comment in when we're using `async`/`await` to indicate that you'll need a browser that supports it, or to use a transpiler like Babel.

Reply:
added async/await to the start part, but not to the creation part. may need guidance on the ctor asyncness. that's gnu to me.

Reply:
Not sure what you mean about ctor asyncness. Constructors can't be asynchronous.

Reply:
@davidfowl highlighted the line creating the connection with the ctor and the .build() method being called. i know how to wrap async around the connection.start() method, which i've done in an upcoming PR. i wasn't clear on what part of the line fowler highlighted i should make async is all.

Reply:
Can you use async await instead of promises or are we not there yet?